source/installation.rst

```diff
@@ -105,6 +105,60 @@ install.
 
          $ pip install ubermag
 
+Installing a calculator
+-----------------------
+
+Ubermag can use different existing micromagnetic calculators (currently OOMMF
+and mumax\ :sup:`3`).
+
+.. tab-set::
+
+   .. tab-item:: OOMMF
+
+      When you install `ubermag` using `conda` OOMMF will be automatically
```

Comment:
We use single back ticks here (`ubermag`) but you double back ticks (``ubermag``) in other rst files. Is that a problem?

Reply:
It should be double back ticks. Thanks for pointing that out (too much Markdown in the last time, I guess).